$NetBSD: patch-lib_dpkg_libdpkg.map,v 1.3 2024/02/17 10:57:05 adam Exp $

On systems without version script support, autoconf doesn't resolve wildcards.

--- lib/dpkg/libdpkg.map.orig	2024-02-17 10:43:39.935868380 +0000
+++ lib/dpkg/libdpkg.map
@@ -185,8 +185,9 @@ LIBDPKG_PRIVATE {
 	fd_write;
 	fd_allocate_size;
 	buffer_digest;
-	buffer_skip_*;
-	buffer_copy_*;
+	buffer_skip_Int;
+	buffer_copy_IntInt;
+	buffer_copy_IntPtr;
 
 	# Subprocess and command handling
 	subproc_signals_ignore;
